src/features/lightspeed/errors.ts

```diff
@@ -184,11 +184,18 @@ ERRORS.addError(
     "Could not find an API Key for IBM watsonx Code Assistant. Please contact your administrator.",
   ),
 );
+ERRORS.addError(
+  403,
+  new Error(
+    "error__wca_organization_not_linked",
+    "Your user Id is not linked to an organization Id. Please contact your administrator.",
+  ),
+);
 ERRORS.addError(
   403,
   new Error(
     "error__wca_model_id_not_found",
-    "Could not find a Model Id for IBM watsonx Code Assistant. Please contact your administrator.",
+    "Your organization does not have an IBM watsonx Code Assistant model configured. Contact your Red Hat organization administrator to configure a model, or provide a model id override in your VSCode Ansible extension settings",
```

Comment:
I know this sentence came straight from the issue I wrote, but... (sorry!)
I'm thinking "VSCode" is redundant and maybe we can make the specific setting more clear. How about "or specify a model in your Ansible extension settings under Lightspeed: Model Id Override."?

Reply:
I agree with you.  I will work on the changes on service part first and then change extension.